Mathlib/Mathport/Syntax.lean

```diff
@@ -253,19 +244,6 @@ syntax (name := rw) "rw " rwRuleSeq : conv
 
 end Conv
 
-syntax (name := unfreezingI) "unfreezingI " tacticSeq : tactic
```

Comment:
None of the I-variants are necessary anymore, they should be translated directly to the I-less equivalent.